### PR TITLE
ticket 4 completed

### PR DESCRIPTION
**Pull request by aliak629918**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -37,3 +38,43 @@ describe("/api/topics", () => {
...
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
+        body.forEach((articles) => {
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
```
Comment (hevdevs) [inline __tests__/app.test.js]: we **don't** want to include our article body in the response for all articles, only when we get to the articles by id endpoint. Just need to be more specific in the columns you are selecting in your SQL statement

File: __tests__/app.test.js [hunk 2]
```diff
@@ -37,3 +38,43 @@ describe("/api/topics", () => {
...
   });
 });
+
+describe("/api/articles", () => {
+  it("200: should respond with an array of article objects with the following properties: author, title, article_id, topic, created_at, votes, comment_count, sorted by created date in decending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
```
Comment (hevdevs) [inline __tests__/app.test.js]: add another lil expect to this test for the length or the response array to be either greater than zero or equal to what we expect to avoid the chance of any false positives in the case of an empty array!

File: controllers/articles.controller.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const { fetchArticles } = require("../models/articles.model");
+
+exports.getArticles = (req, res, next) => {
+  const { sort_by, order, topic } = req.query;
+  fetchArticles(sort_by, order, topic)
+    .then((articles) => {
+      res.status(200).send(articles);
```
Comment (hevdevs) [inline controllers/articles.controller.js]: we **always** want to be sending our response back on an object with a descriptive key in line with JSON syntax. i.e. .send({articles: articles}) (the shorthand for which would be .send({articles}) )

**Review by hevdevs — CHANGES_REQUESTED** [review]
Key functionality is all looking great, just a couple of changes to be made, please throw in another PR once done!
